daemon/daemon/endpoint.go

```diff
@@ -104,7 +78,7 @@ func (d *Daemon) insertEndpoint(ep *endpoint.Endpoint) {
 
 // Sets the given secLabel on the endpoint with the given endpointID. Returns a pointer of
```

Comment:
// Sets the given secLabel on the endpoint with the given endpointID, dockerID or dockerEPID. Returns the cilium endpoint ID if the endpoint was found, 0 otherwise.